documentation/changelog.rst

```diff
@@ -7,24 +7,21 @@ FlexMeasures Changelog
 v0.31.0 | February XX, 2026
 ============================
 
-.. warning:: Upgrading to this version requires running ``flexmeasures db upgrade`` (you can create a backup first with ``flexmeasures db-ops dump``).
-
 New features
 -------------
-* New forecasting API endpoint `[POST] /sensors/(id)/forecasts/trigger <api/v3_0.html#post--api-v3_0-sensors-id-forecasts-trigger>`_ to trigger forecasts on a sensor [see `PR #1813 <https://www.github.com/FlexMeasures/flexmeasures/pull/1813>`_]
 * Make listing public assets on account page optional [see `PR #1872 <https://www.github.com/FlexMeasures/flexmeasures/pull/1872>`_]
 * Added ``root`` and ``depth`` fields to the `[GET] /assets` endpoint for listing assets, to allow selecting descendants of a given root asset up to a given depth [see `PR #1874 <https://www.github.com/FlexMeasures/flexmeasures/pull/1874>`_]
+* Add `included_fields` param to the asset-listing endpoints [see `PR #1884 <https://www.github.com/FlexMeasures/flexmeasures/pull/1884>`_]
```

Comment:
`included_fields` sounds like there might be an `excluded_fields`, too. I suggest using just `fields` (shorter, and no underscores or hyphens always has my preference where possible).

Reply:
Agreed.